src/node.cpp

```diff
@@ -26,56 +26,68 @@
 
 namespace dpaste {
 
-std::uniform_int_distribution<dht::Value::Id> vid_dist;
-std::mt19937_64 rand_;
+Node::Node()
+{
+    dht::crypto::random_device rdev;
+    std::seed_seq seed {rdev(), rdev()};
+    rand_.seed(seed);
+}
+
+std::string Node::paste(dht::Blob&& blob, dht::DoneCallbackSimple&& cb) {
+    auto pin = codeDist_(rand_);
+    std::stringstream ss;
+    ss << std::hex << pin;
+    auto pin_str = ss.str();
+    std::transform(pin_str.begin(), pin_str.end(), pin_str.begin(), ::toupper);
 
-dht::InfoHash Node::paste(dht::Blob&& blob, dht::DoneCallbackSimple&& cb) {
-	auto h = dht::InfoHash::getRandom();
-	paste(h, std::forward<dht::Blob>(blob), std::forward<dht::DoneCallbackSimple>(cb));
-	return h;
+    paste(pin_str, std::forward<dht::Blob>(blob), std::forward<dht::DoneCallbackSimple>(cb));
+    return pin_str;
 }
 
-void Node::paste(dht::InfoHash hash, dht::Blob&& blob, dht::DoneCallbackSimple&& cb) {
-	auto v = std::make_shared<dht::Value>(std::forward<dht::Blob>(blob));
-	v->id = vid_dist(rand_);
```

Comment:
If we use PHT at some point, the PHT code will then abstract handling the value ID.